tools/dependencies/known-oap-backend-dependencies-es7.txt

```diff
@@ -29,14 +29,18 @@ curator-client-4.0.1.jar
 curator-framework-4.0.1.jar
 curator-recipes-4.0.1.jar
 curator-x-discovery-4.0.1.jar
-elasticsearch-7.0.0.jar
-elasticsearch-cli-7.0.0.jar
-elasticsearch-core-7.0.0.jar
-elasticsearch-geo-7.0.0.jar
-elasticsearch-rest-client-7.0.0.jar
-elasticsearch-rest-high-level-client-7.0.0.jar
-elasticsearch-secure-sm-7.0.0.jar
-elasticsearch-x-content-7.0.0.jar
+elasticsearch-7.5.0.jar
+elasticsearch-cli-7.5.0.jar
+elasticsearch-core-7.5.0.jar
+elasticsearch-geo-7.5.0.jar
+elasticsearch-rest-client-7.5.0.jar
+elasticsearch-rest-high-level-client-7.5.0.jar
+elasticsearch-secure-sm-7.5.0.jar
+elasticsearch-x-content-7.5.0.jar
+lang-mustache-client-7.5.0.jar
+mapper-extras-client-7.5.0.jar
+parent-join-client-7.5.0.jar
+rank-eval-client-7.5.0.jar
```

Comment:
You need to update LICENSE including
1. Versions of `lang-mustache-client` and `parent-join-client` 
1. Add `mapper-extras-client` with license as we missed before
1. Add `rank-eval-client` with license as we missed before, both 6.3.2(es6 package) and 7.5.0(you updated)